### DISCUSSION TITLE
Upgrade OpenGL from 3.3 to 4.1

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#18676] [Feature requests]
What would the potential benefits be to upgrading OpenGL from version 3.3 to version 4.1?? Does anyone know if it could be used to improve performance? MacOS supports version 4.1, that's where I got the number from...

**Comment by duncanspumpkin** [comment]
MacOS only supports 3.3 afaik after that they expect you to use metal (Macos proprietary)

**Reply by 733737** [reply]
macos 10.9 and newer supports 4.1
